_episodes/06-removal.md

```diff
@@ -105,14 +105,14 @@ docker rmi <IMAGE ID>
 {: .challenge}
 
 > ## Helpful cleanup commands
-> What is helpful is to have Docker detect and remove unwanted images and containers for you.
+> What is helpful is to have a command to detect and remove unwanted images and containers for you.
 > This can be done with `prune`, which depending on the context will remove different things.
-> - [`docker container prune`](https://docs.docker.com/engine/reference/commandline/container_prune/) removes all stopped containers, which is helpful to clean up forgotten stopped containers.
-> - [`docker image prune`](https://docs.docker.com/engine/reference/commandline/image_prune/) removes all unused or dangling images (images that do not have a tag). This is helpful for cleaning up after builds.
-> - [`docker system prune`](https://docs.docker.com/engine/reference/commandline/system_prune/) removes all stopped containers, dangling images, and dangling build caches. This is very helpful for cleaning up everything all at once.
+> - [`podman container prune`](https://docs.podman.io/en/latest/markdown/podman-container-prune.1.html) removes all stopped containers, which is helpful to clean up forgotten stopped containers.
+> - [`podman image prune`](https://docs.podman.io/en/latest/markdown/podman-image-prune.1.html) removes all unused or dangling images (images that do not have a tag). This is helpful for cleaning up after builds.
+> - [`podman system prune`](https://docs.podman.io/en/stable/markdown/podman-system-prune.1.html) removes all stopped containers, dangling images, and dangling build caches. This is very helpful for cleaning up everything all at once.
```

Comment:
Is there a logic to which links get aliased at the bottom of the page and which are spelled out explicitly each time?

Reply:
I don't know, probably not really... I am doing mostly 1-1 replacement